contrib/hook.json

```diff
@@ -4,9 +4,7 @@
         "path": "/opt/oci-hook-swap.sh"
     },
     "when": {
-        "annotations": {
```

Comment:
Ideally here we match on `io.kubernetes.pod.name` but I was not able to make it work with

```
-        "always": true
+        "annotations": {
+          "^io\\.kubernetes\\.pod\\.name$": "^virt-launcher.*"
```
in `hook.json`